diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4f2cdc2..622f30b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,43 +16,6 @@ pkg_check_modules(DEPS
     json-glib-1.0
     webkitgtk-3.0)
 
-# Indicate is optional, but required for sound menu support
-pkg_check_modules(INDICATE dbusmenu-glib-0.4 indicate-0.7)
-if(${INDICATE_FOUND})
-set(ADD_PKG ${ADD_PKG}
-    Indicate-0.7
-    Dbusmenu-0.4)
-set(ADD_OPTIONS ${ADD_OPTIONS}
-    --define=HAVE_INDICATE)
-message("-- Indicate enabled")
-else()
-message("-- Indicate disabled")
-endif()
-
-# Unity is required for libunity launchers
-pkg_check_modules(UNITY unity)
-if(${UNITY_FOUND})
-set(ADD_PKG ${ADD_PKG}
-                unity)
-set(ADD_OPTIONS ${ADD_OPTIONS}
-        --define=HAVE_UNITY)
-message("-- Unity enabled")
-else()
-message("-- Unity disabled")
-endif()
-
-# Libnotify is not required but for notifications
-pkg_check_modules(LIBNOTIFY libnotify)
-if(${LIBNOTIFY_FOUND})
-set(ADD_PKG ${ADD_PKG}
-                libnotify)
-set(ADD_OPTIONS ${ADD_OPTIONS}
-        --define=HAVE_LIBNOTIFY)
-message("-- Libnotify enabled")
-else()
-message("-- Libnotify disabled")
-endif()
-
 set (CMAKE_C_FLAGS -w)
 set (NORMAL_CFLAGS ${DEPS_CFLAGS} ${INDICATE_CFLAGS} ${DBUS_MENU_CFLAGS} ${UNITY_CFLAGS} ${LIBNOTIFY_CFLAGS})
 set (NORMAL_LIBRARIES ${DEPS_LIBRARIES} ${DBUS_MENU_LIBRARIES} ${INDICATE_LIBRARIES} ${UNITY_LIBRARIES} ${LIBNOTIFY_LIBRARIES})
